### PR TITLE
Adding hometown file for yummytree

### PR DESCRIPTION
**Pull request by yummytree**
@lhalley what kind of food do you like so I can add them to the hometown file?

### CODE REVIEW

File: yummytree-vernonhills.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Vernon Hills, IL
+
+
```
Comment (lhalley) [inline yummytree-vernonhills.md:3]: I really like hamburgers, tacos but also looking for a craft beer

**Review by lhalley — COMMENTED** [review]
Add some restaurants I would like to go and this is ready to ship

File: yummytree-vernonhills.md (hunk 1 above)
Comment (yummytree) [reply yummytree-vernonhills.md:3]: @lhalley all good added the wanted restaurants

**Review by lhalley — APPROVED** [review]
Looks great

**Pull request merged**